docs/analysis-pipeline.rst

```diff
@@ -202,3 +202,50 @@ To generate a mosaic of images, run:
   sg_create_mosaic -i *T1w_RPI_r_flatten.nii.gz -ifolder ~/project/results_multi_20200907/data_processed/ -p sag -col 20 -row 13 -o fig_mosaic_T1w.png
   # Axial views of GRE-T1w data
   sg_create_mosaic -i *acq-T1w_MTS.nii.gz -ifolder ~/spineGeneric_results/data_processed -s _seg -p ax -col 20 -row 13 -o fig_mosaic_GRE-T1w.png
+
+Results
+----------------
+
+
+.. raw:: html
+  :file: _static/csa_t1.html
```

Comment:
This will currently break, until https://github.com/spine-generic/spine-generic/pull/240 is functional. In the meantime, could you try to add an html in [this dedicated repos](https://github.com/spine-generic/website-data) and link the URL here, and see if it works?